src/resources/js/views/tooltip.js

```diff
@@ -117,21 +116,21 @@ tribe.events.views.tooltip = {};
 	 *
 	 * @since TBD
 	 *
+	 * @param {integer} index jQuery.each index param from 'afterSetup.tribeEvents' event
+	 * @param {jQuery} $container jQuery object of view container
+	 * @param {object} data data object passed from 'afterSetup.tribeEvents' event
+	 *
+	 * @return {void}
 	 */
-	obj.initTooltips = function() {
-
-		$( obj.selectors.tooltip ).tooltipster( {
-			interactive: true,
-			theme: [ 'tribe-common', 'tribe-events', 'tribe-events-tooltip-theme' ],
-			functionInit: function( instance, helper ) {
-				obj.onFunctionInit( instance, helper );
-			},
-			functionReady: function( instance, helper ) {
-				obj.onFunctionReady( instance, helper );
-			},
-			functionAfter: function( instance, helper ) {
-				obj.onFunctionAfter( instance, helper );
-			}
+	obj.initTooltips = function( index, $container, data ) {
+		$container.find( obj.selectors.tooltip ).each( function( index, tooltip ) {
+			$( tooltip ).tooltipster( {
+				interactive: true,
+				theme: [ 'tribe-common', 'tribe-events', 'tribe-events-tooltip-theme' ],
+				functionInit: obj.onFunctionInit,
```

Comment:
Did you test this and see if it is working? Or just updated the code?

Reply:
What do you mean @juanfra?

Reply:
That you requested me to change that previously, to have the same format it has here. But when I made those changes it wasn't triggering the tooltip nor the different things we implemented for the accessible tooltip. So, I was wondering if we did some dev QA on these changes or if we'll end up finding out that tooltips are not working next time we want to use them.